lxd/storage/drivers/driver_zfs_utils.go

```diff
@@ -13,10 +13,18 @@ import (
 	"github.com/lxc/lxd/lxd/migration"
 	"github.com/lxc/lxd/shared"
 	"github.com/lxc/lxd/shared/ioprogress"
+	"github.com/lxc/lxd/shared/units"
 )
 
-// zfsBlockVolSuffix suffix used for block content type volumes.
-const zfsBlockVolSuffix = ".block"
+const (
+	// zfsBlockVolSuffix suffix used for block content type volumes.
+	zfsBlockVolSuffix = ".block"
+
+	// zfsMinBlockSize is a minimum value for recordsize and volblocksize properties.
+	zfsMinBlocksize = 512
+	// zfsMinBlockSize is a maximum value for recordsize and volblocksize properties.
+	zfsMaxBlocksize = 1024 * 1024
```

Comment:
Sounds like we'll be bumping that one to 16 * 1024 * 1024 then

Reply:
Fixed